### ISSUE TITLE
pdf2html.pages & pdf2html.html does not return images

### ISSUE THREAD

**Issue by JasaNiklanovic** [shebinleo/pdf2html#44]
Hi, pdf2html.pages & pdf2html.html only returns paragraphs & links, but no images. Is this possible? Are there any workarounds available to get images (in the right order between other textual elements)?

**Comment by shebinleo** [comment]
@JasaNiklanovic Yes, this might be possible. I can look into this. If you are interested please feel free to modify and raise a PR.

**Comment by zileeva** [comment]
Hello @shebinleo !

I'm also experiencing the same problem. Would be great to add it! Thank you!

**Comment by Ali-muhammed7040** [comment]
@shebinleo i want to contribute in this project to fix images issue. When we convert pdf to html it is not handling images

**Comment by shebinleo** [comment]
Added a new feature that will allow to extract images from the PDF. See #44

**Comment by maclonghorn** [comment]
I'm using v 4.4.0 and I'm seeing broken images.  HTML is:
```
<img src="embedded:image1.png" alt="image1.png">
```
